webapp/channels/src/components/advanced_text_editor/message_with_mentions_footer.tsx

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {useIntl} from 'react-intl';
+
+import {InformationOutlineIcon} from '@mattermost/compass-icons/components';
+
+export default function MessageWithMentionsFooter() {
```

Comment:
This seems like a separate change from the one mentioned in the ticket - should we break it out?

Reply:
moved